src/App.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './components/Header.js';
+import Add from './components/Add.js';
+import Delete from './components/Delete.js';
+import Save from './components/Save.js'
+import Titles from './components/Titles.js';
+import Note from './components/Note.js';
+
+class App extends Component {
+	state = {
+		currentNote: {
+			id: null,
+			title: "",
+			text: ""
+		},
+		allNotes: {
+			1: {
+				id: 1,
+				title: "Sometext",
+				text: "here is some text"
+			},
+			2: {
+				id: 2,
+				title: "Moretext",
+				text: "here is more text"
+			}
+		}
+	}
+
+	componentDidMount() {
+		fetch('https://noteworthy-graphql.herokuapp.com/v1/graphql', {
+			method: 'POST',
+			headers: { 'Content-Type': 'application/json' },
+			body: JSON.stringify({
+				query: '{ notes { id title text } }'
+			})
+		})
+		.then(data => data.json())
+		.then(obj => console.log(obj.data.notes[0]))
+	}
+
+	renderTitles = () => {
+		let titlesToBeRendered = Object.values(this.state.allNotes).map(note => 
+			<li onClick={ (event) => this.selectNote(event) } 
+				key={ note.id.toString() }>
+					<input onChange={ (event) => this.saveTitle(event) }
+						className="title-box"  
+						defaultValue={ note.title }
+						id={ note.id }
+					/>
+			</li>
+		);
+		return titlesToBeRendered;
+	}
+
+	renderNote = () => {
```

Comment:
pass the content of the note directly to the component as a prop instead of using a method (see the prop in the render method)